content/glossary/git.md

```diff
@@ -23,3 +20,23 @@ ${type} - Git Object Type as a string
 ${size} - size in bytes of ${content} represented as a string base 10.
 
 ${content} - []byte of the content
+
+# Git Blob
+
+A git `blob` is the type used for file contents in git:
+
+![gitblob](/img/git_blob.svg)
+
+- ${content} - []byte of the file contents
+  - Does not include filename or path
+  - Does not include mode information
+  - Does not include *any* metadata
+  - Just the contents
+  - Any file anywhere with the same contents will have the same ‘blob’ object
+  - Any file anywhere with the same contents will have the same git ref
+
+
+# Git Ref
+
+Git Blobs are identified by the sha1 of the blob object:
```

Comment:
helpful definition, though I would prefer we indicate that we're using the sha1 algorithm for simplicity & compatibility with `git`, and only for that purpose, and would update to any future format if/when git does

Reply:
@AevaOnline Let me know if the addition below works :)